api/tools/src/main/java/org/pmiops/workbench/tools/cdrconfig/UpdateCdrConfig.java

```diff
@@ -166,6 +166,14 @@ private void preCheck(CdrConfigVO cdrConfig) {
           cdrDefaultVersionPerTier.put(v.accessTier, v.cdrVersionId);
         }
       }
+
+      if (!StringUtils.isAllEmpty(v.wgsFilterSetName, v.wgsBigqueryDataset)
+          && StringUtils.isAnyEmpty(v.wgsFilterSetName, v.wgsBigqueryDataset)) {
+        throw new IllegalArgumentException(
+            String.format(
+                "Both wgsBigqueryDataset and wgsFilterSetName have to be defined. Current values are: wgsBigqueryDataset: %s / wgsFilterSetName: %s",
```

Comment:
I'd prefer something like "if one is defined, you must define the other" because this text is not strictly true: **Neither** is also fine.

Reply:
so the json files in this directory get tested automatically in circle? In other words, when I add a new json file, do I need to configure anything to make it run?

Reply:
No, there's no automation connected to these.  But I was able to confirm this works by running:
```
cp tools/src/test/resources/cdrconfig/cdr_wgsBigqueryDataset_not_defined.json config/cdr_config_local.json
 ./project.rb update-cdr-config-local^C
```
```
IllegalArgumentException: If either of wgsBigqueryDataset and wgsFilterSetName is defined, the other has to be defined too. Current values are: wgsBigqueryDataset: null / wgsFilterSetName: example
```
Thanks for the test files!